### ISSUE TITLE
Manifest lookups for DestinyHistoricalStatsDefinition table

### ISSUE THREAD

**Issue by vpzed** [jgayfer/pydest#3]
It looks like the manifest lookup code will not work for the DestinyHistoricalStatsDefinition table since it uses a string based key column instead of the usual signed integer id column.

**Comment by jgayfer** [comment]
Right you are. This check should fix things:
```
if isinstance(hash_id, int):
            hash_id = self._twos_comp_32(hash_id)
```
However I'm having some weird issues with not being able to query the database. I would welcome help!

**Comment by vpzed** [comment]
Did you also update the dbase.py query function which has id hard-coded?

    def query(self, hash_id, definition):
        sql = """
              SELECT json FROM {}
              WHERE id = {}
              """
        self.cur.execute(sql.format(definition,hash_id))
        return self.cur.fetchall()

According to this SQL (so you can check the manifest DB for yourself):

    SELECT name, sql FROM sqlite_master
    WHERE type='table'
    ORDER BY name;

the only table that does not use an integer id column as the "key" is the DestinyHistoricalStatsDefinition table.  So you may wish to add an if/else clause for the _twos_comp_32 and the query function based on the definition value instead of the hash_id type.

**Comment by jgayfer** [comment]
Fixed by f34154cbd0f180c2f20f53a926d3ab603ccb7497

Thank you @vpzed very much for your help. The quotes were the part that was really killing me - thank you for pointing that out!
